ush/get_crontab_contents.py

```diff
@@ -58,7 +58,7 @@ def get_crontab_contents(called_from_cron):
     # themselves being called as cron jobs.  In that case, we must instead
     # call the system version of crontab at /usr/bin/crontab.
     #
-    if MACHINE == "CHEYENNE":
+    if MACHINE == "CHEYENNE" or "DERECHO":
```

Comment:
@natalie-perlin The reason the unit test is failing is that this is not correct python syntax.

```suggestion
    if MACHINE == "CHEYENNE" or MACHINE == "DERECHO":
```